core/Tracker/Settings.php

```diff
@@ -58,6 +61,22 @@ public function getConfigId(Request $request, $ipAddress)
 
         $browserLang = substr($request->getBrowserLanguage(), 0, 20); // limit the length of this string to match db
 
+        if ($this->isSameFingerprintsAcrossWebsites) {
+            $fingerprintSalt = ''; // fingerprint salt won't work when across multiple sites since all sites could have different timezones
```

Comment:
Interesting. be good to add a comment in global.ini.php above the setting like "when you set this to 1, this will also set enable_fingerprinting_limited_when_cookie_disabled=0"

Reply:
that is already done